contrib/babelfishpg_common/sql/geometry.sql

```diff
@@ -338,16 +338,63 @@ CREATE OR REPLACE FUNCTION sys.ST_zmflag(sys.GEOMETRY)
 	AS '$libdir/postgis-3', 'LWGEOM_zmflag'
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
 
-CREATE FUNCTION sys.ST_Equals(leftarg sys.GEOMETRY, rightarg sys.GEOMETRY)
+CREATE OR REPLACE FUNCTION sys.STArea(sys.GEOMETRY)
+	RETURNS float8
+	AS '$libdir/postgis-3','ST_Area'
+	LANGUAGE 'c' IMMUTABLE STRICT;
```

Comment:
is there any specific reason why this is not marked as `PARALLEL SAFE`?

Reply:
Resolved